vg/vgimg/vgimg.go

```diff
@@ -161,7 +166,16 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 		c.img = img
 		c.gc = gc
 		c.dpi = gc.GetDPI()
-		return setsDPI | setsSize
+		return setsDPI | setsSize | setsBackground
+	}
+}
+
+// UseBackgroundColor specifies the image background color.
+// The default color is white.
```

Comment:
`// Without UseBackgroundColor the default color is white.` just to be clear since the context here is a little smeared.

Reply:
fixed